### PR TITLE
DirectDruidClient: Fix division by zero.

### PR DESCRIPTION
**Pull request by gianm**
Introduced in #3954 when some floating math was changed to
integer math. This patch restores the old math.

### CODE REVIEW

**Comment by dclim** [comment]
👍

File: server/src/main/java/io/druid/client/DirectDruidClient.java [hunk 1]
```diff
@@ -269,13 +269,14 @@ public InputStream nextElement()
...
+          final long nodeTimeMs = TimeUnit.NANOSECONDS.toMillis(nodeTimeNs);
           log.debug(
               "Completed queryId[%s] request to url[%s] with %,d bytes returned in %,d millis [%,f b/s].",
               query.getId(),
               url,
               byteCount.get(),
-              TimeUnit.NANOSECONDS.toMillis(nodeTimeNs),
-              byteCount.get() / TimeUnit.NANOSECONDS.toSeconds(nodeTimeNs)
+              nodeTimeMs,
+              byteCount.get() / (0.001 * nodeTimeMs)
```
Comment (jihoonson) [inline server/src/main/java/io/druid/client/DirectDruidClient.java]: It would be better to leave a comment here.

Comment (gianm) [inline server/src/main/java/io/druid/client/DirectDruidClient.java]: Added a comment.

**Comment by jihoonson** [comment]
LGTM!

**Pull request merged**